src/main/java/appeng/core/AEConfig.java

```diff
@@ -99,6 +99,7 @@
 	public double meteoriteClusterChance = 0.1;
 	public double meteoriteSpawnChance = 0.3;
 	public int[] meteoriteDimensionWhitelist = { 0 };
+	public String[] meteoriteValidSpawnBlocks = new String[] {};
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make meteoriteValidSpawnBlocks a static final constant or non-public and provide accessors if needed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)